Dockerfile

```
@@ -0,0 +1,14 @@
+#syntax=docker/dockerfile:1.2
+
+FROM alpine:latest AS luacheck
+
+LABEL maintainer="Caleb Maclennan <caleb@alerque.com>"
+
+RUN apk add --no-cache dumb-init lua lua-argparse lua-filesystem
```

For readability I always prefer to list the intstallable items in a list, like this:

```suggestion
RUN apk add --no-cache \
    dumb-init \
    lua \
    lua-argparse \
    lua-filesystem
```
and to reduce the container size, maybe remove the apk cache as well?

```
     rm -rf "/var/cache/apk/"*
```

---

You don't have to remove the apk cache at all if you tell it `--no-cache`. It does have a directory still, but it is empty. I checked in the built image.